src/stores/game.js

```diff
@@ -160,7 +166,7 @@ export const useGameStore = defineStore('game', {
       return state.turn % 2 === state.myPNum;
     },
     hasGlassesEight: (state) => {
-      return state.player.faceCards.filter((card) => card.rank === 8).length > 0;
+      return state.player?.faceCards?.filter((card) => card.rank === 8).length > 0 ?? null;
```

Comment:
Shouldn’t player always be set? Is there a case where it’s not set that we should be fixing instead of no-oping here?

Reply:
I was getting "can't read .facecards from undefined" on the initial joining of the lobby. I guess because it is checking for it before the players are actually set

Reply:
Sounds like we should be fast failing on that case instead then. If there are many cases checking this where we benefit from the chaining I’m not against setting a logical default, I just want to make sure you’re doing your due diligence in validating.

Reply:
I think fast fail is a good idea, I think it's only checked in two spots